modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryServiceTest.java

```diff
@@ -689,21 +690,35 @@ public void testValidateMaximumNumberOfGuestUserObjectEntriesPerObjectDefinition
 		AssertUtils.assertFailure(
 			ObjectEntryCountException.class,
 			StringBundler.concat(
-				"Unable to exceed ", 1,
-				" guest object entries for object definition ",
-				_objectDefinition.getObjectDefinitionId()),
+				"The limit of guest entries for ", _objectDefinition.getLabel(),
+				" has been reached and will no longer be accepted. Please ",
+				"contact the administrator for further assistance."),
 			() -> _objectEntryService.addObjectEntry(
 				0, _objectDefinition.getObjectDefinitionId(),
 				Collections.emptyMap(),
 				ServiceContextTestUtil.getServiceContext(
 					TestPropsValues.getGroupId(), _guestUser.getUserId())));
 
+		Role adminRole = _roleLocalService.getRole(
+			_objectDefinition.getCompanyId(), RoleConstants.ADMINISTRATOR);
+
+		long[] adminUserIds = _userLocalService.getRoleUserIds(
+			adminRole.getRoleId());
+
+		for (long adminUserId : adminUserIds) {
+			Assert.assertFalse(
+				_userNotificationLocalService.
+					getDeliveredUserNotificationEvents(
```

Comment:
For performance reasons, can we use the correlated method `getDeliveredUserNotificationEventsCount` here?